### ISSUE TITLE
Nullable Guid's don't work

### ISSUE THREAD

**Issue by ShawnTheBeachy** [SlapperAutoMapper/Slapper.AutoMapper#78]
I think the simplest explanation is just a sample:

```
private sealed class Owner
{
    public Guid? PrimaryImageId { get; set; }
}

static void Main (string[] args)
{
    var id = Guid.NewGuid();
    dynamic data = new System.Dynamic.ExpandoObject();
    data.PrimaryImageId = id;

    Owner owner = Slapper.AutoMapper.MapDynamic<Owner>(data, false);
    Console.WriteLine($"PrimaryImageId: {owner.PrimaryImageId}");
}
```

EDIT: Removed some unnecessary parts from the demo.

**Comment by ShawnTheBeachy** [comment]
IMHO this is major. Like, "needs fixed immediately" major. This issue only came to my attention because images started disappearing from my site when an entity was edited, since the `PrimaryImageId` property was set to null when I retrieved it from the database, and was subsequently saved back as null. I imagine there are quite a few other sites/apps that would be affected by this.

**Comment by stefanosandes** [comment]
Any update about this problem?

**Comment by ShawnTheBeachy** [comment]
@stefanosandes I switched to https://github.com/Dolfik1/Mappy due to the lack of action here. Nearly painless and it's been working for me ever since.

**Comment by randyburden** [comment]
The fix for this has been released in [v2.0.2](https://github.com/SlapperAutoMapper/Slapper.AutoMapper/releases/tag/v2.0.2) and pushed to [NuGet](https://www.nuget.org/packages/Slapper.AutoMapper/2.0.2)
